source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,169 @@
+ To prevent users from logging in again, use a RADIUS server for authentication with appropriately configured limits. See :ref:`label-radius-server`
+
+Authentication methods
+----------------------
+
+Four authentication methods are avaliable on pfSense :
+
+:Don't authenticate users: All users will be redirected to the captive portal
+  login page but will get connected as soon as they click **Login**. this is useful for
+  displaying a welcome page, or forcing users to see general terms and conditions.
+:Use an authentication server from the user manager: Authentication will be made using a remote authentication server. Please see :ref:`label-user-manager` for details.
+:Use RADIUS MAC Authentication: This authentication method emulate 802.1X, allowing some MAC addresses to get automatically connected.
+:Use vouchers: Vouchers are one-time use portal acces code. Please see :doc:`captive portal vouchers <captive-portal-vouchers>` page for details.
+
+
+Vouchers can be used in parallel with any authentication method, allowing for example
+login/password authentication for employees, and voucher auth for others users.
+
+
+.. _label-user-manager:
+
+Use an authentication server from the user manager
+--------------------------------------------------
+
+It is possible to authenticate users against a server from the server manager, such as an LDAP or RADIUS server.
+Please check the user manager documentation for details on how to setup remote authentication servers in pfSense.
+
+Remote authentication server have to be setup first in the server manager before being able to use it for captive portal authentication.
+
+
+Special note about "unauthenticated" username
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using an authentication server, it is recommanded to block/reserve the username "unauthenticated", in order to prevent users to connect using this username.
+This can be done by registering a disabled account in the remote authentication server (recommanded when using LDAP) or by preventing this user to login (recommanded when using a RADIUS server).
+
+This username is used internally in pfSense for defining an user not yet authenticated, or for defining connected users when using "no authentication" method and no username has been provided. For this reason, this username is given special rights, such as bypassing "block concurrent logins" option.
+
+
+.. _label-radius-server:
+
+Authenticating captive portal users using a RADIUS server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using an external RADIUS server (such as FreeRADIUS) to authenticate users, it is possible to set
+some attributes in the RADIUS Access-Accept response that will be understood by pfSense,
+in order to fine-tune the how the captive portal will will behave for each user.
+
+.. tip:: pfSense's RADIUS dictionary may not be included by default in your RADIUS server configuration.
```

Comment:
Don't make `pfSense` possessive. Change `pfSense's RADIUS dictionary` to `The RADIUS dictionary for pfSense`
https://docs.netgate.com/pfsense/en/latest/references/style-guide.html#what-to-avoid